vm/inc/ubpf.h

```diff
@@ -81,8 +89,18 @@ int ubpf_load(struct ubpf_vm *vm, const void *code, uint32_t code_len, char **er
  */
 int ubpf_load_elf(struct ubpf_vm *vm, const void *elf, size_t elf_len, char **errmsg);
 
-uint64_t ubpf_exec(const struct ubpf_vm *vm, void *mem, size_t mem_len);
+uint64_t ubpf_exec(const struct ubpf_vm *vm, void *mem, size_t mem_len, char **errmsg);
```

Comment:
Would it make sense to use the same error_printf function in the interpreter that the jitted program would use?

Reply:
Great suggestion. Made that change.